tools/benchmark_tool/setup.py

```diff
@@ -6,92 +6,18 @@
 """
 Use this script to create a wheel with OpenVINO™ Python* tools:
 
-$ python setup.py sdist bdist_wheel
+$ python -m pip wheel .
 """
-import pkg_resources
-import re
 from setuptools import setup, find_packages
-from pathlib import Path
-from typing import Dict, List
 
 
+with open('requirements.txt', 'r', encoding='utf-8') as f:
```

Comment:
looks like we don't need setup.py here anymore, I would recommend to start using pyproject.toml and drop setup.py at all.

Reply:
Already planned in CVS-156008.